extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -352,7 +357,18 @@ export class AmpStory360 extends AMP.BaseElement {
     }
 
     const container = this.element.ownerDocument.createElement('div');
-    this.canvas_ = this.element.ownerDocument.createElement('canvas');
+    const altText = dev()
+      .assertElement(
+        this.element.querySelector('amp-img') ||
+          this.element.querySelector('amp-video')
+      )
+      .getAttribute('alt-text');
+    this.canvas_ = createElementWithAttributes(
+      this.element.ownerDocument,
+      'canvas',
+      dict({'role': 'img', 'aria-label': altText})
```

Comment:
Is there a `role="video"` or similar?

Reply:
> Is there a role="video" or similar?

There isn't. I think `img` is the closest item from the [list of roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques).

> Also, I wonder if alt-text is already read by screen readers or not, becuase we don't want to read that twice.

Do you mean read once from the original media element, then again from the canvas? 
We hide the original media element. The screen reader does not have access to it which is why we need to put it on the canvas.
I added a comment in the code to help explain that.